decentraland/unity-renderer
# Feat: worlds skybox modifier v1

## What does this PR change?
Closes #3562

## How to test the changes?
1. Goto /changerealm mgoldman.dcl.eth
2. Observe that skybox is changed to 10 AM - visually 
3. Observe that Settings for skybox in the UI Settings are non-interactable (grayed out) and fixedHours set to 10:00

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
